Monday Milestones/numbers1.js

```diff
@@ -0,0 +1,22 @@
+let num1 = 5;
+let num2 = 3;
+console.log(num1 + num2); //sum
```

Comment:
For the sake of your future self, I'd also try to make the output when you run this file as helpful as possible. Right now it will print out a list of numbers when you run it.

Instead, I suggest something like:

```javascript
let num1 = 5;
let num2 = 3;

console.log('--- Numbers!');
console.log('The value of num1 is:', num1);
console.log('The value of num2 is:', num2);
console.log();

console.log('Their sum is:', num1 + num2);
console.log('Their difference is:', num1 - num2);
console.log('Their product is:', num1 * num2);
// etc.
```

Reply:
Think of it as a design exercise if that helps. Imagine you're designing a tiny, interactive demonstration. You want the connection between what is output when you run the file and the lines of code in the file to be as clear as possible.

A nice side effect is that it makes it a more useful reference for your future self, too.